packages/runner/src/utils/collect.ts

```diff
@@ -46,7 +46,7 @@ export function interpretTaskModes(suite: Suite, namePattern?: string | RegExp,
 }
 
 function getTaskFullName(task: TaskBase): string {
-  return `${task.suite ? `${getTaskFullName(task.suite)} ` : ''}${task.name}`
+  return `${task.suite?.name ? `${getTaskFullName(task.suite)} ` : ''}${task.name}`
```

Comment:
What if suites suite name exist, but this one doesn't have it? It should check the result of the function here

Reply:
Yes, Thanks for your review!

Reply:
I meant you should do `getTaskFullName` first, and check against it

Reply:
Yes, I know what you mean. The end result is the same. Because this function ends up returning `test.suite.name`

Reply:
But it leaks implementation details of the function

Reply:
Go it